tracing-opentelemetry/src/integrationTest/java/com/couchbase/client/tracing/opentelemetry/OpenTelemetryIntegrationTest.java

```diff
@@ -72,11 +71,51 @@ static void beforeAll() {
   static void afterAll() {
     cluster.disconnect();
     environment.shutdown();
-    tracerProvider.shutdown();
   }
 
   @Test
   void capturesTraceSpans() {
+    Span parent = otelTesting.getOpenTelemetry().getTracer("integrationtest")
+        .spanBuilder("test")
+        .setSpanKind(SpanKind.SERVER)
+        .startSpan();
+    try (Scope ignored = parent.makeCurrent()) {
+      collection.get("myid");
+    } catch (DocumentNotFoundException ignored) {
+      // expected
+    } finally {
+      parent.end();
+    }
+
+    waitUntilCondition(() -> {
+      otelTesting.assertTraces()
+          .hasTracesSatisfyingExactly(
+              trace -> trace.hasSpansSatisfyingExactly(
+                  span -> span
+                      .hasName("test")
+                      .hasKind(SpanKind.SERVER),
+                  span -> span
+                      .hasName("get")
+                      .hasParentSpanId(parent.getSpanContext().getSpanId())
+                      .hasKind(SpanKind.INTERNAL)
```

Comment:
So we're not really allowed to have nested `CLIENT` spans because the definition in the spec says a single `CLIENT` span corresponds to a single `SERVER` span. In our instrumentation, we generally suppress transport spans, so for example in AWS SDK instrumentation, we generate a `CLIENT` span for the SDK call, but no span for the Apache HTTP Client request itself. In this case, I think that would be similar to suppressing the `dispatch_to_server` span. However, it's not entirely bad modeling to have `dispatch_to_server` as an `INTERNAL` span either, but we'd generally only expect the "API Method" to be a `CLIENT` span - imagine it as the name on client and server should be very similar.  Ideally some day Couchbase itself has OpenTelemetry server instrumentation too and would have a `get` span with type `SERVER` :)

Reply:
One more question: how do you deal with "multiple" requests? So for example we have "get from replicas" which will internally send N requests to the active and all replicas. Would the outer parent span that covers all the individual requests also be a CLIENT? or really only ever the individual requests